samples/bot-configuration-app/csharp/Bot configuration/Bots/TeamsBot.cs

```diff
@@ -0,0 +1,286 @@
+                                ""weight"": ""bolder""
+                            },
+                            {
+                                ""type"": ""Image"",
+                                ""url"": ""data:image/png;base64," + imageData + @""",
+                                ""size"": ""auto""
+                            }
+                          ],
+                ""fallbackText"": ""This card requires Adaptive Card support.""
+            }";
+
+            var attachment = new Microsoft.Bot.Schema.Attachment()
+            {
+                ContentType = AdaptiveCard.ContentType,
+                Content = JsonConvert.DeserializeObject(adaptiveCardJson)
+            };
+
+            var reply = MessageFactory.Attachment(attachment);
+            await turnContext.SendActivityAsync(reply, cancellationToken);
+        }
+
+        protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
+        {
+            if (turnContext.Activity.Text != null)
+            {
+                string text = turnContext.Activity.Text.ToLower().Trim();
+                if (text == "chosen flow" || text == "<at>typeahead search adaptive card</at> chosen flow")
+                {
+                    await turnContext.SendActivityAsync($"Bot configured for {_chosenFlow} flow", cancellationToken: cancellationToken);
+                }
+            }
+            else if (turnContext.Activity.Value != null)
+            {
+                var choiceselect = turnContext.Activity.Value;
+                await turnContext.SendActivityAsync($"Selected option is: {choiceselect}", cancellationToken: cancellationToken);
+            }
+        }
+
+        protected override Task<ConfigResponseBase> OnTeamsConfigFetchAsync(ITurnContext<IInvokeActivity> turnContext, JObject configData, CancellationToken cancellationToken)
+        {
+            /*
+            Option 1: You can add a "config/auth" response as below code
+            Note: The URL in value must be linked to a valid auth URL which can be opened in a browser. This code is only representative and not a working example.
+            */
+            /*ConfigResponseBase response = new ConfigResponse<BotConfigAuth>
+            {
+                Config = new BotConfigAuth
+                {
+                    SuggestedActions = new SuggestedActions
+                    {
+                        Actions = new List<CardAction>
+                {
+                    new CardAction
+                    {
+                        Type = "openUrl",
+                        Title = "Sign in to this app",
+                        Value = "https://example.com/auth"
+                    }
+                }
+                    },
+                    Type = "auth"
+                }
+            };*/
+
+            ConfigResponseBase response = adaptiveCardForContinue();
```

Comment:
mark this is option 2, as there is an option 1 above.

Reply:
Fixed